html/index.html

```diff
@@ -353,39 +356,53 @@ <h1>Sampling and Sequencing Simulator</h1>
 <script type=module>
 import poisson from 'https://cdn.jsdelivr.net/gh/stdlib-js/random-base-poisson@esm/index.mjs';
 import normal from 'https://cdn.jsdelivr.net/gh/stdlib-js/random-base-normal@esm/index.mjs';
+import lognormal from 'https://cdn.jsdelivr.net/gh/stdlib-js/random-base-lognormal@esm/index.mjs';
+
+const epsilon = 0.000001;
 
 function get_input_cv(input, input_cv) {
   const mean = get_input(input);
   const cv = get_input(input_cv) / 100;
 
-  if (cv < 0.1) {
+  if (cv < epsilon) {
     return mean;
   }
 
   const stdev = cv * mean;
   return normal(mean, stdev);
 }
 
-function get_inputs_log_cv(input, input_log_cv) {
-  const means = get_inputs(input);
-  const cv_of_log = get_input(input_log_cv) / 100;
-  if (cv_of_log < 0.01) {
-    return means;
+function get_input_geom_cv(input, input_geom_cv) {
+  const geom_mean = get_input(input);
+  const geom_cv = get_input(input_geom_cv) / 100;
+
+  if (geom_cv < epsilon) {
+    return geom_mean;
+  }
+
+  const geom_stdev = geom_cv * geom_mean;
+  return lognormal(geom_mean, geom_stdev);
+}
+
+function get_inputs_biased(input, input_geom_stdev) {
+  const empirical_values = get_inputs(input);
+  const geom_stdev = get_input(input_geom_stdev) / 100;
+  if (geom_stdev < epsilon) {
+    return empirical_values;
   }
 
-  // Don't want to bias each mean independently, bias all of them by
-  // the CV together.  Also do it in log space, since we want log
-  // normal here.
+  // Don't want to bias each mean independently, bias all of them
+  // together with log-normally distributed noise.  The geometric mean
+  // (and median) of the noise is zero, and the standard deviation is
+  // provided by the user.
 
-  const stdev_of_log = cv_of_log;
-  const bias_of_log = normal(1, stdev_of_log);
+  const bias = Math.exp(normal(0, geom_stdev));
```

Comment:
It's equivalent, but you could use `lognormal(0, geom_stdev)` here to match above.

Reply:
done